src/ripple/consensus/Validations.h

```diff
@@ -302,6 +311,14 @@ class Validations
         beast::uhash<>>
         byLedger_;
 
+    //! Validations from listed nodes indexed by ledger seq (partial and full)
+    beast::aged_unordered_map<
+        Seq,
+        hash_map<NodeID, Validation>,
+        std::chrono::steady_clock,
+        beast::uhash<>>
+        bySeq_;
```

Comment:
Consider this tabbing
```
beast::aged_unordered_map<
        Seq,
        hash_map<NodeID, Validation>,
        std::chrono::steady_clock,
        beast::uhash<>>
    bySeq_;
```

Reply:
The current version is how #2204 formats it. I haven't found a way to get your suggested style yet, but will update it if I can.